src/sectionTitle.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+
+import SectionTitle, { SECTIONTITLE_CLASS } from './SectionTitle';
+
+describe('SectionTitle', function() {
+	let sectionTitle,
+		titleNode;
+
+	beforeEach(() => {
+		sectionTitle = TestUtils.renderIntoDocument(<SectionTitle />);
+		titleNode = ReactDOM.findDOMNode(sectionTitle);
+	});
+	afterEach(() => {
+		sectionTitle = null;
+		titleNode = null;
+	});
+
+	it('exists', function() {
+		expect(titleNode).not.toBeNull();
```

Comment:
Offering the same comment here as in another PR

We have been having discussions on using findDOMNode in some PR's so not gonna mark this as a change request but... offering this as an alternative approach to writing these tests. You can also read more about this [http://reactkungfu.com/2015/07/approaches-to-testing-react-components-an-overview/](here) 

```
it("exists", () => {
    expect(()=>TestUtils.findRenderedDOMComponentWithTag(sectionTitle, SectionTitle)).toThrow();
});
it("renders a list in a box with proper CSS classes", function() {
    let box = this.TestUtils.findRenderedDOMComponentWithTag(this.component, "div");
    expect(box.props.className).toEqual("invitation-list-container");

    let list = this.TestUtils.findRenderedDOMComponentWithTag(box, "ul");
    expect(list.props.className).toEqual("invitation-list");
  });
```